baselines/dom.generated.d.ts

```diff
@@ -10807,7 +10808,7 @@ interface Node extends EventTarget {
     /**
      * Returns the parent element.
      */
-    readonly parentElement: HTMLElement | null;
+    readonly parentElement: Element | null;
```

Comment:
With this in place, the property on `HTMLElement` can be deleted.

Reply:
Please read the entire thread and  #1151 before repeating comments.